bot/src/main/java/com/community/tools/service/github/GitHubConnectService.java

```diff
@@ -34,12 +34,23 @@ public GitHub getGitHubConnection() {
    * @return GHRepository
    */
   public GHRepository getGitHubRepository() {
+    return getGitHubRepositoryByName(nameRepository);
+  }
+
+  /**
+   * Method to get GitHub repository by its name.
+   *
+   * @param repositoryName name of the desired repository
+   * @return GHRepository
+   */
+  public GHRepository getGitHubRepositoryByName(String repositoryName) {
     GHRepository repository;
     try {
-      repository = getGitHubConnection().getRepository(nameRepository);
+      repository = getGitHubConnection().getRepository(repositoryName);
```

Comment:
Try placing return statement here, so there is no need in repository variable.

Reply:
Removed variable to return repository on the spot